src/include/common.h

```diff
@@ -319,9 +319,9 @@ struct _MetaFrameBorders
 void meta_frame_borders_clear (MetaFrameBorders *self);
 
 /* should investigate changing these to whatever most apps use */
-#define META_DEFAULT_ICON_SIZE 48
-#define META_MIN_ICON_SIZE 6
-#define META_MAX_ICON_SIZE 480
+#define META_DEFAULT_ICON_SIZE 64
```

Comment:
It's better to keep `#define META_DEFAULT_ICON_SIZE 48` for non HiDPI resolutions.

Reply:
What resolution do you use? I tested on both HiDPI and regular, but now that I think of it I only tried one resolution (1920x1080, or 3840x2160 for HiDPI). I'll test all possible resolutions tomorrow to make sure things work

Reply:
I used 2560x1600 for HiDPI and on 1920x1200 for regular.